doc/vaulted-shell.1.md

```diff
@@ -10,7 +10,8 @@ SYNOPSIS
 --------
 
 `vaulted shell` *name* [*OPTIONS*]  
-`vaulted shell --assume` *arn* [*OPTIONS*]
+`vaulted shell --assume` *arn* [*OPTIONS*]  
+`vaulted shell --refresh` *name* [*OPTIONS*]
```

Comment:
We probably don't need to call out each individual option (especially since this can be combined with things like `--assume`), `--assume` is just called out specifically because it is nearly its own feature. :)